README.md

```diff
@@ -30,14 +30,8 @@ The simple "standalone" server can be built with:
 ## Using
 
 TLS certificates for the server can be generate by using the helper script
-`contrib/mk-tls-keypair.sh`. Your factory's PKI directory  was generated
-with a `create_ca` script. Once a CA is created, you can upload/authorize
-it with:
-```
- $ fioctl keys ca show --just-device-cas > /tmp/cas.pem
- $ cat <new-ca.pem> >> /tmp/cas.pem
- $ fioctl keys ca update /tmp/cas.pem
-```
+`contrib/mk-tls-keypair.sh`. The `<pki-dir>` parameter is your factory's PKI directory that was generated
```

Comment:
It looks simpler from end user perspective and it worked well for me.
Also, I am not sure that advising to run `create_ca` is correct at this context since it should have been run by `fioctl keys ca create` command and `factory_ca.*` files already exist at this point the pki directory.
We may suggest to run `create_device_ca` to create another device CA in addition to the existing `local-ca` and `online`. However, it's not necessary.